src/components/presentational/Calendar/Calendar.tsx

```diff
@@ -38,7 +46,7 @@ export default function (props: CalendarProps) {
 				days: []
 			};
 			for (let j = 0; j < 7; j++) {
-				if (i === 0 && j < firstDay) {
+				if (i === 0 && j < firstDay + daysOfTheWeekIndeces[j]) {
```

Comment:
Not sure if this is where it is, but it does go into an infinite loop it seems if a number more than 6 is supplied.  Of course if you're using typescript it would complain about that, but probably good to protect against that and default to 0 or something if a value > 6 is supplied